examples/README.md

```diff
@@ -0,0 +1,27 @@
+<!--
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2015 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Examples
+Example package contains a collection of snap examples and [videos](./videos.md) that demonstrate the snap libraries and usages.
+
+* [configs](./configs) is an example of the global configuration file that powers your plugins
+* [influxdb-grafana](./influxdb-grafana) has an example of [snap Influxdb publisher plugin](https://github.com/intelsdi-x/snap-plugin-publisher-influxdb). The example demonstrates
```

Comment:
There's also one that is for psutil only without pcm. You can also link the pcm and psutil plugins. I'd capitalize Grafana too since they do. 
```
The following scripts can be used to start the demos
run-pcm.sh <docker-machine name>
starts snapd
creates and starts a task that collects from both the pcm and psutil plugins
run-psutil.sh <docker-machine>
starts snapd
creates and starts a task that collects from psutil plugin
./run.sh <docker-machine>
configured influxdb and grafana only
```